### DISCUSSION TITLE
Error "make:migration" command not found after upgrade

### DISCUSSION THREAD

**Discussion by yarapolana** [adonisjs/core#1833] [Help] [answered]
I have upgraded the adonis app to the latest version (october release) and everything is working fine except the migrations commands, they all seem to have disapeared.

Has something change?

This is what comes up with `node ace --help`

<img width="710" alt="Screenshot 2020-11-02 at 04 29 44" src="https://user-images.githubusercontent.com/19730118/97871856-f6102b80-1d0c-11eb-8179-b23d9ec0e54a.png">

**Comment by dawidow** [comment, marked as answer]
Hey, I think you have to install **lucid**. Use `npm i @adonisjs/lucid@alpha` and then invoke it with `node ace invoke @adonisjs/lucid`. Why I think it should work? Because you have neither migrattion command nor model command. I hope it’ll work.

Lucid comes to you with Database and also new commands like make:migration or make:model. 

Probably you lost this package in update process.

**Reply by yarapolana** [reply]
Hey @dawidow 
Thanks for the reply, I made sure to reinstall and invoke it twice before raising the issue, the version I got is `lucid 9.0.3` and unfortunately, the command doesn't seem to register.

I did though uninstalled and installed everything again (core, lucid, auth...), and now only when invoking lucid I get this error `RangeError: Maximum call stack size exceeded`, the db installation and instructions work fine but the command doesn't get registered.

I created a new project to simulate an upgrade, everything works fine until you eventually upgrade the packages, maybe something on the way gets lost, I am using yarn.

**Reply by thetutlage** [reply]
Have you followed the [upgrade steps](https://preview.adonisjs.com/blog/october-2020-release#steps-to-upgrade) mentioned in release blog post? We also published a screencast on same https://www.youtube.com/watch?v=6MBNw6JOaYQ

**Reply by yarapolana** [reply]
ahhhhhhh... I missed this line
```
export default listDirectoryFiles(__dirname, Application.appRoot, [
  './commands/index.js',
])

```

**Reply by hakkesshu2020** [reply]
Thanks, dawidow.
This happens on the latest clean install with `npm init adonis-ts-app blog`. I had to run your ace invoke command to get it to work.

Ah! I forgot there was a separate setup for database. :/
